doc/sphinx-guides/source/installation/config.rst

```diff
@@ -801,7 +809,8 @@ List of S3 Storage Options
     dataverse.files.<id>.label                   <?>                 **Required** label to be shown in the UI for this storage                   (none)
     dataverse.files.<id>.bucket-name             <?>                 The bucket name. See above.                                                 (none)
     dataverse.files.<id>.download-redirect       ``true``/``false``  Enable direct download or proxy through Dataverse.                          ``false``
-    dataverse.files.<id>.upload-redirect         ``true``/``false``  Enable direct upload of files added to a dataset  to the S3 store.          ``false``
+    dataverse.files.<id>.upload-redirect         ``true``/``false``  Enable direct upload of files added to a dataset in the S3 store.           ``false``
+    dataverse.files.<id>.upload-out-of-band      ``true``/``false``  Enable direct upload of files added to a dataset via API only.              ``false``
```

Comment:
"Allow upload of files by out-of-band methods (using some tool other than Dataverse)"?